src/platform/utilities/feature-toggles/flipper-client.js

```diff
@@ -106,4 +104,7 @@ function makeFlipperClient() {
 
   return flipperClientInstance;
 }
-export { makeFlipperClient as FlipperClient };
+
+module.exports = {
```

Comment:
this is the only other change needed to make the `FlipperClient` work both in the browser and in node.